### ISSUE TITLE
Do not hide yourself

### ISSUE THREAD

**Issue by eklaburn1** [r-a-y/bp-hide-user#2]
Doesn't hide admins from buddy press Members list

**Comment by eklaburn1** [comment]
Hello,

I wanted to say I have your plugin installed and it does indeed work, but it doesn't allow you to hide yourself...this means that I had to give another account admin access temporarily to hide the main admin account. This was fine and dandy until I revoked admin status from that temporary account, as the hide admin function done by that account was also revoked along with it. So when an admin hides an account, then is downgraded from admin status, any accounts he hid are visible again. 

Is there anyway you can update this to allow admins to hide themselves? Or make it so that hides are permanent (but can be undone by other admins) even if the admin is deleted or demoted? 

Thank you!

**Comment by r-a-y** [comment]
> So when an admin hides an account, then is downgraded from admin status, any accounts he hid are visible again.

This is not true, the plugin does not record who hid a user so what you describe cannot happen.

**Comment by eklaburn1** [comment]
Hello Ray,

I have no idea how its happening, but it's happening. So I made a screen record video to show it:

https://youtu.be/6oL1_Hqdw5w

Thanx for the reply!
